mwdb/web/src/components/RichAttribute/MarkedMustache.tsx

```diff
@@ -135,6 +146,9 @@ class MustacheContext extends Mustache.Context {
             if (!query) return undefined;
             return new SearchReference(query, currentObject);
         }
+        if (isFunction(currentObject)) {
+            currentObject = currentObject.call(this.view);
```

Comment:
I see that object returned by lookup is already called by Mustache.js as a part of `renderSection`. Do we need to resolve it at lookup level?

https://github.com/janl/mustache.js/blob/master/mustache.js#L609